### DISCUSSION TITLE
please add the following codes to the organisation-role-service codelist

### DISCUSSION THREAD

**Discussion by gabrieledall** [OP-TED/eForms-SDK#99] [Ideas]
the following codes cpb-aq and cpb-awa shoud be added to the organisation-role-service since these two organisation roles also provide service to others. 
In germany, these two codes are required and as they already belong to the parent codelist "organisation" it shoud be easy to add them.

**Comment by YvesJo** [comment]
The identification of the CPB Awarding/Acquiring role of an organization is performed at the organization level using the fields:

- Awarding CPB Buyer Indicator (OPP-051-Organization)
- Acquiring CPB Buyer Indicator (OPP-052-Organization)

The "organisation-role-service" is restricted to economic operators acting as Procurement Service Providers for public organizations.

**Reply by gabrieledall** [reply]
Hi Yves,

thank you for explaining these two fields to me. I found all the other roles but couldn't find these two.

Yours
Gabriele
